docs/tutorials/oss-to-enterprise-migration-new.md

```diff
@@ -0,0 +1,99 @@
+# Upgrading from Open Source to Enteprise Using a New Server
+
+Congratulations, you have decided to use Rundeck Enterpise! You are on you way to automating your workflows. Before that happens, you need to get your Rundeck Enterprise set up. Fear not, this guide will get you set up no matter what kind of Rundeck user you are. 
+
+## RPM Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
+* Export project setting from Rundeck Open Source. Go to the project that you wish to export, go to project settings and select "Export Archive." On the next page, you will be able to choose what all you want exported (Jobs, Executions, Configurations, etc.)
```

Comment:
Isn't there a link or page about exporting projects out there?

Reply:
Yes, should I link it in that step?